journal-of-the-marine-biological-association-of-the-united-kingdom.csl

```diff
@@ -0,0 +1,599 @@
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never">
```

Comment:
@rafazildod, this style should probably be fixed to British English (`default-locale="en-GB"`), see https://github.com/citation-style-language/styles/wiki/Style-Requirements#3---style-locale.